components/WaveHandPicker.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { useState } from 'react';
+import { WaveHandButton } from '@/components/WaveHandButton';
+import {HandWaveBadge } from '@/lib/draw_badge_api';
+
+const waveHandEmoji = '👋';
+
+interface WaveHandPickerProps {
+  currentWaveHands: string[];
```

Comment:
I think the argument to this component is likely more of a `defaultHands` or `initialHands` rather than than `currentHands` if the evolution of this component is like the content of `NameTagForm`

Reply:
Changed to `initialHands`.